src/screens/ProfileScreen/index.js

```diff
@@ -48,9 +48,14 @@ import {
   updateBioProfile,
   updateImageProfile
 } from '../../service/profile';
-import {deleteAnonymousPost, deletePost} from '../../service/post';
+import {deleteAnonymousPost, deletePost, viewTimePost} from '../../service/post';
 import {downVote, upVote} from '../../service/vote';
-import {DEFAULT_PROFILE_PIC_PATH, SOURCE_MY_PROFILE} from '../../utils/constants';
+import {
+  DEFAULT_PROFILE_PIC_PATH,
+  NavigationConstants,
+  SOURCE_FEED_TAB,
+  SOURCE_MY_PROFILE
+} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `viewTimePost` function and `NavigationConstants` are newly imported. Ensure that these functions are properly tested and their usage doesn't introduce any side effects in the existing functionality.
```diff
+ import {deleteAnonymousPost, deletePost, viewTimePost} from '../../service/post';
+ import {
+   DEFAULT_PROFILE_PIC_PATH,
+   NavigationConstants,
+   SOURCE_FEED_TAB,
+   SOURCE_MY_PROFILE
+ } from '../../utils/constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->